### ISSUE TITLE
Support for the Telldus Pool Thermometer

### ISSUE THREAD

**Issue by janhoy** [nlrb/com.weather-sensors#94]
I bought this https://www.kjell.com/no/produkter/smarte-hjem/smarte-sensorer/smarte-temperatursensor/telldus-bassengtermometer-for-tellstick-p50435 and tried to discover it though your Homey app, but does not show up.

See guide: https://telldus.com/en/wp-content/uploads/sites/2/2020/10/313291_Manual-1.pdf
Other shop: https://shop.smarthome-europe.com/en/peripheriques/4556-telldus-poolspa-thermometer-7330985132919.html

**Comment by janhoy** [comment]
I purchased a SDR and ran `rtl_433` and found my device, identified as (with `-a 4` argument):
```
time      : 2022-06-15 22:06:01
model     : Fineoffset-WH2                         ID        : 199
Temperature: 28.1 C      Integrity : CRC
*** signal_start = 2169515, signal_end = 2211239, signal_len = 41724, pulses_found = 48
Iteration 1. t: 237    min: 115 (30)    max: 359 (18)    delta 8
Iteration 2. t: 237    min: 115 (30)    max: 359 (18)    delta 0
Pulse coding: Short pulse length 115 - Long pulse length 359

Short distance: 249, long distance: 0, packet distance: 250

p_limit: 237
bitbuffer:: Number of rows: 48 
[00] { 1} 00        : 0
[01] { 1} 00        : 0
[02] { 1} 00        : 0
[03] { 1} 00        : 0
[04] { 1} 00        : 0
[05] { 1} 00        : 0
[06] { 1} 00        : 0
[07] { 1} 00        : 0
[08] { 1} 80        : 1
[09] { 1} 00        : 0
[10] { 1} 80        : 1
[11] { 1} 80        : 1
[12] { 1} 00        : 0
[13] { 1} 00        : 0
[14] { 1} 80        : 1
[15] { 1} 80        : 1
[16] { 1} 80        : 1
[17] { 1} 00        : 0
[18] { 1} 00        : 0
[19] { 1} 00        : 0
[20] { 1} 80        : 1
[21] { 1} 80        : 1
[22] { 1} 80        : 1
[23] { 1} 00        : 0
[24] { 1} 80        : 1
[25] { 1} 80        : 1
[26] { 1} 80        : 1
[27] { 1} 00        : 0
[28] { 1} 00        : 0
[29] { 1} 80        : 1
[30] { 1} 80        : 1
[31] { 1} 00        : 0
[32] { 1} 00        : 0
[33] { 1} 00        : 0
[34] { 1} 00        : 0
[35] { 1} 00        : 0
[36] { 1} 00        : 0
[37] { 1} 00        : 0
[38] { 1} 00        : 0
[39] { 1} 00        : 0
[40] { 1} 00        : 0
[41] { 1} 80        : 1
[42] { 1} 80        : 1
[43] { 1} 00        : 0
[44] { 1} 00        : 0
[45] { 1} 80        : 1
[46] { 1} 00        : 0
[47] { 1} 80        : 1
```

Now, will this device already be supported? I have re-tried detecting it as a "Temperature" unit in Weather-Sensors app, by pushing the reset button right before pushing "Connect" button, but it always comes up empty "No new units found".

**Comment by janhoy** [comment]
Found this blog with the deciphering of Fineoffset-WH2: https://lucsmall.com/2012/04/29/weather-station-hacking-part-2/
<img width="640" alt="Skjermbilde 2022-06-16 kl  01 27 28" src="https://user-images.githubusercontent.com/409128/173958706-1fb7d71d-217e-4704-83a7-dd0ff0df5d04.png">
Looks like `rtf_433` output got the bits switched around, so 0 is 1 and 1 is 0, that gives this decoding
```
preable  device-id    temp         humid%   crc
11111111 010011000111 000100011001 11111111 10011010
                      281*10=28.1
```
And voila - `000100011001` translates to 281 which meand 28.1 ºC

**Comment by janhoy** [comment]
Found it. I had misunderstood how your app discovers new devices. Now that I went into app settings and enabled all protocols, I saw a signal appearing for *Alecto V3*, and then I got the `WH2` device configured, which is my pool:
![IMG_8505](https://user-images.githubusercontent.com/409128/173960109-875829e0-c2e8-48a4-ad7a-d1ed181fbee9.jpg)

Perhaps the docs at https://homey.app/no-no/app/com.weather-sensors/Wireless-Weather-Sensors/ could be more precise on how to configure the app?

PS: This GitHub repo links to an apps.athom.com URL that no longer works, it should be https://homey.app/no-no/app/com.weather-sensors/Wireless-Weather-Sensors/

I'll close this issue now since there is nothing to do.
